python/sglang/srt/server_args.py

```diff
@@ -264,10 +264,11 @@ def add_cli_args(parser: argparse.ArgumentParser):
             help="How conservative the schedule policy is. A larger value means more conservative scheduling. Use a larger value if you see requests being retracted frequently.",
         )
         parser.add_argument(
+            "--tensor-parallel-size",
             "--tp-size",
             type=int,
             default=ServerArgs.tp_size,
-            help="The tensor parallelism size.",
+            help="Number of tensor parallel replicas.",
```

Comment:
Why change this description? And if changes are needed here, why hasn't the `dp` been changed?